main-page-witout-login.html

```diff
@@ -0,0 +1,310 @@
+                    </div>
+                    <div class="section__content-block col-md-6 text-right">
+                        <div class="section__content">
+                            <div class="section__title">
+                                Track your statistics and set new goals
+                            </div>
+                            <div class="section__text">
+                                BookLover will tell you more about your book habits ..
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </section>
+        </div>
+    </div>
+
+    <footer class="footer container-fluid">
+        <div class="footer__content container">
+            <div class="footer__row row">
+                <div class="footer__social col-lg-4 col-sm-12">
+                    <a href="#" class="footer__social-icon social-network">
+                        <i class="social-network__icon fa fa-instagram fa-2x"></i>
+                    </a>
+                    <a href="#" class="footer__social-icon social-network">
+                        <i class="social-network__icon fa fa-facebook-f fa-2x"></i>
+                    </a>
+                    <a href="#" class="footer__social-icon social-network">
+                        <i class="social-network__icon fa fa-vk fa-2x"></i>
+                    </a>
+                    <a href="#" class="footer__social-icon social-network">
+                        <i class="social-network__icon fa fa-twitter fa-2x"></i>
+                    </a>
+                </div>
+                <div class="footer__stores col-lg-4 col-sm-12">
+                    <div class="social-btn social-btn--light">
+                        <a class="social-btn__link" href="#">
+                            <i class="social-btn__icon fa fa-apple fa-lg"></i>
+                            <div class="social-btn__text-block">
+                                <div class="social-btn__text">Get it on</div>
+                                <div class="social-btn__store">App Store</div>
+                            </div>
+                        </a>
+                    </div>
+                    <div class="social-btn social-btn--light">
+                        <a class="social-btn__link" href="#">
+                            <i class="social-btn__icon fa fa-google fa-lg"></i>
+                            <div class="social-btn__text-block">
+                                <div class="social-btn__text">Get it on</div>
+                                <div class="social-btn__store">Google Play</div>
+                            </div>
+                        </a>
+                    </div>
+                </div>
+                <div class="footer__copyright col-lg-4 col-sm-12">
+                    © 2020, BookLover
+                </div>
+            </div>
+        </div>
+    </footer>
+
+
+
+<script src="assets/js/script.js"></script>
```

Comment:
I believe this should be loaded after all the others so it has access to jquery and such no?

Reply:
Done